{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "NONE",
   "body" : "### Describe the issue\r\nAfter importing a private key from an old wallet, the rescan tool a long time, but after completion all Timestams belonging to transactions related to the newly imported key are wrong.\r\n#### What behavior did you expect?\r\nShow the correct timestamps of all trasactions, corresponding to the block the transaction took place.\r\n#### What was the actual behavior (provide screenshots if the issue is GUI-related)?\r\nsee screenshots. The newly imported key has the label \"ehemals\", The big number for confirmations (in the tooltip) seems to be correct, but the timestamps all show the time of the latest transaction (lieferando) which was there at the time of import.\r\n#### How reliably can you reproduce the issue, what are the steps to do so?\r\nHave not reproduced this issue yet (because it takes sooo long). But I have observed other strange things before (data curruption, see: Issue https://github.com/bitcoin/bitcoin/issues/11627).\r\n### What version of Bitcoin Core are you using, where did you get it (website, self-compiled, etc)?\r\nBitcoin core from the ppa used on Ubuntu 17.4 (most recent, I think) Bitcoin core version: v0.15.0.0-g3751912e8e (64-Bit)\r\n### What type of machine are you observing the error on (OS/CPU and disk type)?\r\nUbuntu 17.04 64 bit, Quadcore CPU, 2GB RAM, HDD and SSD (for chainstate).\r\n### Any extra information that might be useful in the debugging process.\r\nEverything else seems normal. The blockchain was freshly reindexed the 3 days before the import.(to cure the data corruption). The former issue \"data corruption\" was observed on a different computer, I then used a desktop computer. But I use the same (extermal) HDD.\r\n",
   "closed_at" : "2022-11-22T22:32:47Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=4",
      "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
      "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
      "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
      "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/MarcoFalke",
      "id" : 6399679,
      "login" : "MarcoFalke",
      "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
      "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
      "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
      "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/MarcoFalke"
   },
   "comments" : 17,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703/comments",
   "created_at" : "2017-11-16T07:19:54Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/11703",
   "id" : 274417401,
   "labels" : [
      {
         "color" : "08a781",
         "default" : false,
         "description" : null,
         "id" : 149424,
         "name" : "Wallet",
         "node_id" : "MDU6TGFiZWwxNDk0MjQ=",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Wallet"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "MDU6SXNzdWUyNzQ0MTc0MDE=",
   "number" : 11703,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "state_reason" : "completed",
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703/timeline",
   "title" : "Strange state (Timestamps wrong) after importing a priv key.",
   "updated_at" : "2022-11-22T22:32:47Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/11703",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/15250834?v=4",
      "events_url" : "https://api.github.com/users/kollokollo/events{/privacy}",
      "followers_url" : "https://api.github.com/users/kollokollo/followers",
      "following_url" : "https://api.github.com/users/kollokollo/following{/other_user}",
      "gists_url" : "https://api.github.com/users/kollokollo/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/kollokollo",
      "id" : 15250834,
      "login" : "kollokollo",
      "node_id" : "MDQ6VXNlcjE1MjUwODM0",
      "organizations_url" : "https://api.github.com/users/kollokollo/orgs",
      "received_events_url" : "https://api.github.com/users/kollokollo/received_events",
      "repos_url" : "https://api.github.com/users/kollokollo/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/kollokollo/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/kollokollo/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/kollokollo"
   }
}
